docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/addDocument/fields/formula/Formula.rst

```diff
@@ -0,0 +1,146 @@
+   :width: 600
+   :align: center
+
+3. Select formula field
+
+.. image:: pic_formula/Screenshot_11.png
+   :width: 600
+   :align: center
+
+4. Select the SUM function, in the arguments field, select a numeric and currency fields
+
+.. image:: pic_formula/Screenshot_12.png
+   :width: 600
+   :align: center
+
+4. Save the formula field. Fill in the main fields of  template. Create a draft envelope
+
+5. Filling numeric and currency field
+
+.. image:: pic_formula/Screenshot_3.png
+   :width: 600
+   :align: center
+
+6.After filling fields, pay attention to  formula field
+
+.. image:: pic_formula/Screenshot_4.png
+   :width: 600
+   :align: center
+
+7. Send envelope
+
+In order to apply the SUM table column function, you must repeat the following steps:
+
+1. Create Template
+
+2. Go to template create  dynamic table, added numeric field to dynamic table
+
+.. image:: pic_formula/Screenshot_6.png
+   :width: 600
+   :align: center
+
+3. Select formula field,select the SUM table column, in the arguments field, select a numeric field.
+
+.. image:: pic_formula/Screenshot_122.png
+   :width: 600
+   :align: center
+
+4. Save the formula field. Fill in the main fields of  template. Create a draft envelope
+
+5. Add two rows to table, filling numeric field
+
+.. image:: pic_formula/Screenshot_8.png
+   :width: 600
+   :align: center
+
+6. After filling fields, pay attention to formula field
+
+.. image:: pic_formula/Screenshot_9.png
+   :width: 600
+   :align: center
+
+7. Send envelope
+
+!!!For users who will work through integrations, it is necessary to understand that the SUM table column function contains one attribute,
+   and looks like this: <formula precision="2" name="total">SUM({rowTotal})</formula>, but the SUM function has two arguments
+
+   .. image:: pic_formula/Screenshot_10.png
+      :width: 600
+      :align: center
+
+
+In order to apply the COUNTA table column function, you must repeat the following steps:
+
+1. Create Template
+2. Create dynamic table field
+3. Add number field in table
+4. Select formula field, adding argument
+
+.. image:: pic_formula/Screenshot_56.png
+   :width: 600
+   :align: center
+
+5. Fill in the main fields of template
+6. Create envelope
+7. Add rows to table. and fill number fields
+
+.. image:: pic_formula/Screenshot_01.png
+   :width: 600
+   :align: center
+
+8. COUNTA table column function shows number of rows in the table.
+
+In order to apply the MIN/MAX table column function, you must repeat the following steps:
+
+1. Create Template
+2. Create dynamic table field
+3. Add number field and currency field in table
+4. Select MIN table column formula, adding argument
+
+.. image:: pic_formula/Screenshot_02.png
+     :width: 600
+     :align: center
+
+5. Select MAX table column function, adding argument
+
+.. image:: pic_formula/Screenshot_03.png
+   :width: 600
+   :align: center
+
+6. Fill in the main fields of template
+7. Create envelope
+8. Add rows to table. and fill number fields
```

Comment:
remove dot before "...and fill..."

it's better to write "fill in" instead of "fill"